Assets/MRTK/Examples/Common/Profiles/RiggedHands/RiggedHandControllerVisualizationProfile.asset

```diff
@@ -24,8 +24,8 @@ MonoBehaviour:
     type: 3}
```

Comment:
There is a setting to stop the joints from rendering with the hand mesh.

In the Hand Tracking section in the Input profile:

![image](https://user-images.githubusercontent.com/53493796/99435432-fb50a700-28c4-11eb-9556-26060f986037.png)

Change `Hand Joint Visualization Modes` to `Nothing` instead of `Everything`

Reply:
Lingering changes from testing, will revert

Reply:
Leaving the joint visualization in the editor because it shows the origin of the grab/hand pointer and is generally useful.